libraries/DataFlash/DataFlash_File.cpp

```diff
@@ -197,6 +199,7 @@ void DataFlash_File::periodic_1Hz(const uint32_t now)
         _write_fd = -1;
         _initialised = false;
     }
+    df_stats_log();
```

Comment:
Should this be inside an else? I mean, what is the point of writing the stats if the IO thread is dead?

Reply:
Still nice to know what's going on.  It's not going to hurt anybody and is slightly easier to read (IMHO)

Reply:
I'm not sure how you will know what's going on if it isn't written into the log, but I'm not fussed about this.